cockroachdb/src/jepsen/cockroach/multiregister.clj

```diff
@@ -110,12 +110,12 @@
     {:name        "multi-register"
      :client      {:client (MultiAtomicClient. (atom false) nil)
                    :during (independent/concurrent-generator
-                            (count (:nodes opts))
+                            3
```

Comment:
Why is this 3 instead of the size of the cluster like it was before?

Reply:
Reducing the concurrency sped up the test. Setting this to the size of the cluster was somewhat arbitrary. Tests like `comments` do that but tests like `register` set it to a constant value.